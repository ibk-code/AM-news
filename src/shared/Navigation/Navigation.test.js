import { shallow } from 'enzyme';
import React from 'react';
import Navigation from './index';

describe('Navigation Component test', () => {
  it('expect to render Navigation  component', () => {
    expect(shallow(<Navigation />)).toMatchSnapshot();
  });
});
