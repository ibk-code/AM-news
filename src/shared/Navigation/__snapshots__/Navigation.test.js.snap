// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Component test expect to render Navigation  component 1`] = `
<Fragment>
  <div
    className="nav-wrap shadow"
  >
    <Container
      fluid={false}
    >
      <h1>
        <Link
          className="orange"
          to="/"
        >
          Beta
          <small>
            News
          </small>
        </Link>
      </h1>
    </Container>
  </div>
</Fragment>
`;
